packages/nimble-components/src/table/tests/table-keyboard-navigation.spec.ts

```diff
@@ -652,6 +617,54 @@ describe('Table keyboard navigation', () => {
                 expect(pageObject.getCell(0, 0).menuOpen).toBe(true);
             });
 
+            it('when a cell action menu is opened via keyboard, scrolling the table will close the action menu and focus the cell', async () => {
```

Comment:
Can you double check that we have a test somewhere that ensures the action menu is closed when data is updated via `setData`?

Reply:
Added that test.